fortio_main.go

```diff
@@ -536,9 +537,23 @@ func grpcClient() {
 		}
 		return
 	}
-	_, err := fgrpc.PingClientCall(host, count, httpOpts.PayloadUTF8(), *pingDelayFlag, &httpOpts.TLSOptions)
+	_, err := fgrpc.PingClientCall(host, count, httpOpts.PayloadUTF8(), *pingDelayFlag, &httpOpts.TLSOptions, grpcMD())
 	if err != nil {
 		// already logged
 		os.Exit(1)
 	}
 }
+
+func grpcMD() map[string][]string {
+	md := make(map[string][]string)
+	headers := bincommon.SharedHTTPOptions().ExtraHeaders
+	for k, v := range headers {
+		k = strings.ToLower(k)
+		if k == "user-agent" {
```

Comment:
what happens if that's set is grpc meta data?

Reply:
As explained above, and additional work may be required to handle the `host` header separately.